shutdown.if

```diff
@@ -91,6 +91,24 @@ interface(`shutdown_signal',`
 
 ########################################
 ## <summary>
+##     Send SIGCHLD signals to shutdown.
+## </summary>
+## <param name="domain">
+##     <summary>
+##     Domain allowed access.
+##     </summary>
+## </param>
+#
+interface(`shutdown_sigchld',`
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Mon, Oct 19, 2015 at 01:08:10PM -0700, Jason Zaman wrote:
> > @@ -91,6 +91,24 @@ interface(`shutdown_signal',`
> >  
> >  ########################################
> >  ## <summary>
> > +##     Send SIGCHLD signals to shutdown.
> > +## </summary>
> > +## <param name="domain">
> > +##     <summary>
> > +##     Domain allowed access.
> > +##     </summary>
> > +## </param>
> > +#
> > +interface(`shutdown_sigchld',`
> 
> why is this here? it seems unrelated to all the other changes

This is important. basically when you shutdown then shutdown_t becomes
pid 1

and since everyone needs to be able to send sighld to pid 1 they need to
also be able to send sigchld to shutdown_t.

thus this should be called in the domain.te file as:

tunable_systemd
    shutdown_sigchld(domain_t)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/TresysTechnology/refpolicy-contrib/pull/4/files#r42419143

- -- 
02DFF788
4D30 903A 1CF3 B756 FB48  1514 3148 83A2 02DF F788
https://sks-keyservers.net/pks/lookup?op=get&search=0x314883A202DFF788
Dominick Grift
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQGcBAEBCgAGBQJWJVBNAAoJENAR6kfG5xmcRv0L/RYjCzOCV2glYm66hDC3940D
f659R1+Wi2tIAdmtkFXvFvHxj3w7eLqCoC0GoWJ7VjzSN1ZV5zxAT4rnWoiYx5NW
jcirX8sJz39XUW4LSgEo5tREcfgG4Lhfl7R0aWqDJzYZiUuVgFNbXm/1bsizcqlE
S228W1q0qkKubxNmPlKdI5YB02s8+2wftpAAJ0+CXWSrtOoMlrvn/yexMf+LA2Z2
v6k1xGF3Db3uHmIHAqo1NHDggAtgaQbB0Lax8PocR7Shuc/CkbA9EeGKq6uwf3Fz
W2BAb19rdYHqjgWjaxGk/EyoVxC88s6c6gSWyqUwZIVnVEQxdTpgFnq3Zls8gwSV
yT8gGKmNXTmJOY2+iZRueXjwQcGVDohUa3PxuSliaB5ryqr21N50qPf6d26MqI9q
X23Y3zKvzV6/k9dPnSMMpClhHcJ4SEozdujByvXQXehtUNUbXAZ904YLEpvb0gj/
YLfSIAmq10f+ce2/g9J7RZuGC2hEGHRkJWJQBexO9A==
=m77f
-----END PGP SIGNATURE-----